pkg/block/fetcher.go

```diff
@@ -533,6 +534,23 @@ func contains(s1 []ulid.ULID, s2 []ulid.ULID) bool {
 	return true
 }
 
+type ReplicaLabelsFilter struct {
+	ReplicaLabels []string
+}
+
+func (f *ReplicaLabelsFilter) Filter(metas map[ulid.ULID]*metadata.Meta, synced GaugeLabeled, view bool) {
+	for u, meta := range metas {
+		labels := meta.Thanos.Labels
+		for _, replicaLabel := range f.ReplicaLabels {
+			if _, exists := labels[replicaLabel]; exists {
+				delete(labels, replicaLabel)
+				synced.WithLabelValues(replicaExclude).Inc()
```

Comment:
This is wrong then I think? At least not consistent with other filters.

Usually we add some label like this and increment when we filter block.

Reply:
Anything that is not filtered, will be incremented by len(not filtered) {loaded}

Reply:
Maybe a separate metrics here?